tests/data/IgnoredGetter.graphqls

```diff
@@ -2,6 +2,7 @@ type IgnoredGetter {
   public: String!
   publicWithArgs(arg1: String!, arg2: Int!, arg3: [String] = ["foo"]): [String]!
   isValid: Boolean!
+  isIsser: Boolean!
```

Comment:
This seems to be redundant with `isValid`.

Reply:
I used a field which was not otherwise used for a specific matching unit test.